### PR TITLE
ENT-RMP-195 Fix table header sort icons and behavior

### PR DESCRIPTION
**Pull request by rtamarasinghe**
- Added icons to component
- Prevent scrollbars appearing in sort dropdown
- Update list column sort when user sorts list (earlier was only looking at initial definition in list.json)
- Story changes
    Changed initialSortOrder of columns in list.json
    Commented out unavailable function getCandidates()

### CODE REVIEW

File: storybook/stories/core/components/List/List.story.vue [hunk 1]
```diff
@@ -781,7 +781,7 @@ export default defineComponent({
           );
           const query = [`orderField=${orderField}`, `orderBy=${value}`];
           let includes;
-          getCandidates(appliedFilters.value, includes, query);
+          // getCandidates(appliedFilters.value, includes, query);
```
Comment (lgtkaushalya) [inline storybook/stories/core/components/List/List.story.vue:784]: Why is this comentedout?

Comment (rtamarasinghe) [reply storybook/stories/core/components/List/List.story.vue]: Have removed the commented out code

File: storybook/stories/core/components/List/List.story.vue [hunk 2]
```diff
@@ -871,4 +870,4 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-</style>
\ No newline at end of file
+</style>
```
Comment (Super-Chama) [inline storybook/stories/core/components/List/List.story.vue:873]: Please add new line at end

**Review by Super-Chama — COMMENTED** [review]
image
Visually the icon appear in black. i think we should add some styles to change color to text color.

File: components/src/core/components/Icon/icons.ts [hunk 3]
```diff
@@ -332,6 +332,24 @@ export const oxdNoDataFound: icon = {
...
...32-183.638 191.147-3.414 12.288 7.509 39.594 20.48 47.787 4.096 2.73 17.067 5.462 27.99 4.779 17.067 0 41.643-20.48 153.6-129.024l133.12-128.341 2.048 382.975c1.365 344.064 2.73 383.658 12.971 395.264 13.653 15.018 51.882 16.385 66.219 1.365 6.144-5.461 11.605-10.923 12.288-11.605s3.414-173.397 5.461-382.976l3.414-381.611 130.389 126.976c135.168 131.754 146.091 139.264 180.223 120.832 16.385-8.874 22.529-37.547 12.288-57.343-4.779-8.192-87.381-92.843-184.32-187.050-144.042-141.994-178.176-172.032-193.878-172.032s-49.834 30.037-198.655 178.858z"></path><path fill="#000" d="M543.402 591.189c-6.144 5.462-11.606 10.923-12.288 11.606-1.365 0.683-3.414 173.397-5.462 382.976l-3.414 381.611-130.389-126.976c-111.957-109.227-133.803-127.659-151.553-127.659-35.499 0-57.343 34.134-40.96 63.487 4.779 8.874 87.381 93.525 184.32 187.734 144.042 141.995 178.176 172.032 193.878 172.032s49.834-30.037 198.656-178.859c98.986-98.986 181.59-184.32 183.637-191.147 3.414-12.288-7.509-39.594-20.48-47.787-4.096-2.73-16.385-5.461-27.99-4.779-17.067 0-41.643 20.479-153.6 129.024l-133.12 128.341-2.048-382.976c-1.365-344.064-2.73-383.658-12.971-395.264-13.653-15.018-51.882-16.385-66.219-1.365z"></path></svg>\n'
```
Comment (Super-Chama) [inline components/src/core/components/Icon/icons.ts:338]: might have to put `style="fill:currentColor"` to match the text color

File: storybook/stories/core/components/List/List.story.vue (hunk 2 above)
Comment (rtamarasinghe) [reply storybook/stories/core/components/List/List.story.vue:873]: added

File: components/src/core/components/Icon/icons.ts (hunk 3 above)
Comment (rtamarasinghe) [reply components/src/core/components/Icon/icons.ts]: added this to the svg

File: storybook/stories/core/components/List/List.story.vue (hunk 2 above)
Comment (rtamarasinghe) [reply storybook/stories/core/components/List/List.story.vue:873]: added

**Pull request merged**